es-rest-client-1.3/core/src/main/java/com/bazaarvoice/elasticsearch/client/core/util/ExceptionUtil.java

```diff
@@ -0,0 +1,47 @@
+package com.bazaarvoice.elasticsearch.client.core.util;
```

Comment:
I don't think this is the right path. Knee-jerk thinking... Maybe `org.elasticsearch.ElasticsearchExceptionHelper`?